react/fields/Dropdown/Dropdown.js

```diff
@@ -178,7 +137,7 @@ export default class Dropdown extends Component {
 
     return (
       <div className={classNames}>
-        {this.renderLabel()}
+        <FieldLabel {...this.props} />
```

Comment:
I guess the middle ground here is we pluck the props that the `FieldLabel` cares about and it should still work, right?